tests/adapters/test_protocol.py

```
@@ -39,15 +39,15 @@ def adapter():
                                     ([1, 2, 3, 4], b"\x01\x02\x03\x04"),
                                     (5, b"5"),
                                     (4.6, b"4.6"),
-                                    (None, None),
                                     ))
 def test_to_bytes(input, output):
     assert to_bytes(input) == output
 
 
-def test_to_bytes_invalid():
+@mark.parametrize("input", (5.5j, None))
+def test_to_bytes_invalid(input):
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)